src/components/Data/Documents/Page.vue

```diff
@@ -584,6 +584,32 @@ export default {
         otherQueryParams
       )
       this.$router.push({ query: mergedQuery })
+    },
+    addHumanReadableDateFields() {
+      const keys = []
+      const findDateFields = (mapping, previousKey) => {
+        for (let key in mapping) {
```

Comment:
Use a `for (const .. of ...)` loop instead (https://alligator.io/js/for-of-for-in-loops/)

Reply:
I need the key, if I use a for of i can't access to the key right ?
The site say :
Use for…in to iterate over the properties of an object (the object keys):
Could you explain me ?

Reply:
Oh yes sorry, usually we do this `for (const key of Object.keys(mapping))` but I agree that `for (let ... in ...)` work pretty the same here.  
The only differences are kind of nitpicking:
 - `const` instead of `let` ensure your can't reassign variable
 - more explicit iteration over keys (IMHO)
 - consistency between all for loop

Reply:
Thanks, i'll do the changes :+1: